addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -2037,6 +2037,82 @@ QUnit.module("Views", (hooks) => {
         assert.verifySteps(["create"]);
     });
 
+    QUnit.test(
+        "save a quick create record and create a new record at the same time",
+        async (assert) => {
+            serverData.views["partner,some_view_ref,form"] =
+                "<form>" + '<field name="foo"/>' + '<field name="int_field"/>' + "</form>";
```

Comment:
This is old style, we can use template strings now :-)

Reply:
the test can be simplified, I don't think you need that view (simply remove `quick_create_view` attr)